File: _정답코드/1주차/08_최빈값_찾기/solution.py
def solution(numbers):
    freq = {}
    for num in numbers:
        freq[num] = freq.get(num, 0) + 1

    max_freq = 0
    result = 0

    for num, count in freq.items():
        if count > max_freq:
            max_freq = count
            result = num

    return result

File: _정답코드/1주차/08_최빈값_찾기/solution.cpp
#include <vector>
#include <string>
#include <map>
using namespace std;

int solution(vector<int> numbers) {
    map<int, int> freq;
    for (int i = 0; i < numbers.size(); i++) {
        int num = numbers[i];
        freq[num]++;
    }
    
    int maxfreq = 0, result = 0;
    for (auto& [k, v] : freq) {
        if (v > maxfreq) {
            maxfreq = v;
            result = k;
        } else if (v == maxfreq && result > k) {
            result = k;
        }
    }
    return result;
}